src/main/java/seedu/address/logic/commands/view/ViewNoteCommand.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.logic.commands.view;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.commands.results.CommandResult;
+import seedu.address.model.Model;
+import seedu.address.model.note.Note;
+
+/**
+ * View a specified note.
+ */
+public class ViewNoteCommand extends ViewCommand {
+
+    public ViewNoteCommand(Index targetIndex) {
+        super(targetIndex);
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
+        if (targetIndex.getZeroBased() >= model.getNoteListSize()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_INDEX);
+        }
+
+        Note toView = model.getNote(targetIndex);
+        model.updateFilteredNoteList(Model.PREDICATE_SHOW_ALL_NOTES);
```

Comment:
This will reset the list if the note list was "finding" something, I'm not sure whether you mean to do this.